.github/workflows/snap.yaml

```diff
@@ -21,3 +26,38 @@ jobs:
         with:
           name: snap
           path: ${{ steps.build-rockcraft.outputs.snap }}
+
+      - name: Get branch name
+        id: vars
+        run: echo ::set-output name=short_ref::${GITHUB_REF#refs/*/}
```

Comment:
This is now deprecated and should thus be replaced by 

`echo "short_ref=${GITHUB_REF#refs/*/}" >> "$GITHUB_OUTPUT"`

Reply:
your optimization looks like the right approach! thanks